### PR TITLE
Fix Checkbox Position

### PR DESCRIPTION
**Pull request by kre8sions**
Resolving issue here: https://github.com/powerhome/playbook/issues/341

utilizing Icon and Body Kits within the checkbox component

### CODE REVIEW

File: app/pb_kits/playbook/pb_checkbox/_checkbox.html.erb [hunk 1]
```diff
@@ -5,5 +5,7 @@
   <span class="pb_checkbox_checkmark">
     <%= pb_rails("icon", props: { icon: "check", id: "check_icon", classname: "check_icon", fixed_width: true}) if object.icon %>
   </span>
-  <span class="pb_checkbox_label"><%= object.text %></span>
+  <span class="pb_checkbox_label">
+    <%= pb_rails("body", props: { text: object.text, dark: object.dark ? true : false }) %>
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_checkbox/_checkbox.html.erb]: `dark` is already a boolean field.

Comment (garettarrowood) [inline app/pb_kits/playbook/pb_checkbox/_checkbox.html.erb]:
```suggestion
    <%= pb_rails("body", props: { text: object.text, dark: object.dark }) %>
```

File: app/pb_kits/playbook/pb_checkbox/_checkbox.jsx [hunk 2]
```diff
@@ -13,29 +14,25 @@ type CheckboxProps = {
...
     > <input type="checkbox" name={name} value={value} defaultChecked={checked}/>
   <span className="pb_checkbox_checkmark">
-      <i className="far fa-check check_icon"></i>
+      <Icon className="check_icon" icon="check" fixedWidth />
     </span>
-    <span className="pb_checkbox_label">
-      {label}
-    </span>
-    </Tag>
+    <Body className={(dark === true ? '_dark' : '') + ` pb_checkbox_label`}  >{label}</Body>
```
Comment (drborges) [inline app/pb_kits/playbook/pb_checkbox/_checkbox.jsx]: How about:

```jsx
<Body className={`${dark ? '_dark' : '' } pb_checkbox_label `}>{label}</Body>
```

Alternatively, using `classnames`:

```jsx
const bodyClassName = {
  'pb_checkbox_label': true,
  '_dark': dark,
}

...

<Body className={bodyClassName}>{label}</Body>
```

File: app/pb_kits/playbook/pb_checkbox/_checkbox.jsx [hunk 3]
```diff
@@ -13,29 +14,25 @@ type CheckboxProps = {
 
 const Checkbox = ({
   dark=false,
-  tag='label',
-  label,
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_checkbox/_checkbox.jsx]: The props on the rails kit are:

:dark 
:checked
:icon
:text
:value
:name

Shouldn't they match in the react kit?

Comment (garettarrowood) [reply app/pb_kits/playbook/pb_checkbox/_checkbox.jsx]: The ones in here look to be:

:dark   
:name   
:value   
:checked
:label   

So its missing `icon` and `text` but contains a different prop called `label`.

**Pull request merged**